PhysicsTools/NanoAOD/python/particlelevel_cff.py

```diff
@@ -89,17 +89,6 @@
     src = rivetLeptonTable.src,
 )
 
```

Comment:
Code suggestion to rename generator-level FiducialMET, so that it can be used for MET studies and Rivet-compatible unfolding.

```suggestion

rivetMetTable = simpleSingletonCandidateFlatTableProducer.clone(
    src = cms.InputTag("particleLevel:mets"),
    name = cms.string("FiducialMET"),
    doc = cms.string("MET from Rivet-based ParticleLevelProducer in fiducial volume abs(eta)<5"),
    variables = cms.PSet(PTVars),
)

```

Reply:
Hi Markus, Very right. Thank you. Let take those suggestions to the commit.